frontend/src/common/analytics/events.ts

```diff
@@ -64,11 +65,14 @@ export enum EVENTS {
   DOWNLOAD_DATA_COMPLETE = "DOWNLOAD_DATA_COMPLETE",
   DOWNLOAD_DATA_COPY = "DOWNLOAD_DATA_COPY",
   DOWNLOAD_DATA_FORMAT_CLICKED = "DOWNLOAD_DATA_FORMAT_CLICKED",
-  HOMEPAGE_LEARN_FIND_SINGLE_CELL_DATA_CLICKED = "HOMEPAGE_LEARN_FIND_SINGLE_CELL_DATA_CLICKED",
-  HOMEPAGE_LEARN_EXPLORE_GENE_EXPRESSION_CLICKED = "HOMEPAGE_LEARN_EXPLORE_GENE_EXPRESSION_CLICKED",
-  HOMEPAGE_LEARN_ANALYZE_DATASETS_CLICKED = "HOMEPAGE_LEARN_ANALYZE_DATASETS_CLICKED",
-  HOMEPAGE_LEARN_DOWNLOAD_DATA_CLICKED = "HOMEPAGE_LEARN_DOWNLOAD_DATA_CLICKED",
-  HOMEPAGE_LEARN_EXPEDITE_COLLABORATION_CLICKED = "HOMEPAGE_LEARN_EXPEDITE_COLLABORATION_CLICKED",
+  HOMEPAGE_SEC_NAV_CENSUS = "HOMEPAGE_SECONDARY_NAV_CENSUS_CLICKED",
```

Comment:
I think the key and value need to be the same, otherwise Amanda will get `HOMEPAGE_SECONDARY_NAV_CENSUS_CLICKED` when she wants `HOMEPAGE_SEC_NAV_CENSUS`!